plugins/woocommerce/src/Internal/Admin/Settings/PaymentProviders/WooPayments/WooPaymentsService.php

```diff
@@ -717,15 +587,15 @@ public function get_onboarding_kyc_session( string $location, array $self_assess
 		);
 
 		if ( is_wp_error( $account_session ) ) {
-			throw new Exception( esc_html( $account_session->get_error_message() ), intval( esc_attr( $account_session->get_error_code() ) ) );
+			throw new Exception( esc_html( $account_session->get_error_message() ) );
```

Comment:
Do not include the error code since that is most often a string and Exception wants numbers.